governance.md

```diff
@@ -0,0 +1,145 @@
+
+#### List of peers
+- Rachel Andrew (@rachelandrew)
+- Daniel Beck (@ddbeck)
+- Ryan Johnson (@escattone), Mozilla
+- Jean-Yves Perrier (@teoli2003), Mozilla
+- Eric Shepherd (@a2sheppy), Mozilla
+- Irene Smith (@irenesmith), Mozilla
+- Estelle Weyl (@estelle), Mozilla
+- John Whitlock (@jwhitlock), Mozilla
+
+A Peer who shows an above-average level of contribution to the project, particularly with respect to its strategic direction and long-term health, may be nominated to become an Owner, described below.
+
+### Owners
+The mdn-browser-compat-data project is jointly governed by the [Mozilla MDN staff team](https://wiki.mozilla.org/Engagement/MDN_Durable_Team#Team_Members), the [MDN Product Advisory Board Members](https://developer.mozilla.org/en-US/docs/MDN/MDN_Product_Advisory_Board/Members), and the [Owner group](#list-of-owners). They are collectively responsible for high-level guidance of the project.
+
+The [Owner group](#list-of-owners) has final authority over this project including:
+
+- Technical direction of the project, especially infrastructure PRs and linting and/or schema decisions.
+- Project governance and process (including this policy).
+- Contribution policy.
+- GitHub repository hosting.
+
+Being an Owner is not time-limited. There is no fixed size of the Owner group. The Owner group should be of such a size as to ensure adequate coverage of important areas of expertise balanced with the ability to make decisions efficiently.
+An Owner may be removed from the Owner group by voluntary resignation, or by a standard Owner group motion.
+
+Changes to the Owner group should be posted in the agenda, and may be suggested as any other agenda item (see [Project Meetings](#project-meetings) below).
+
+Owners have additional responsibilities over and above those of a Peer. These responsibilities ensure the smooth running of the project. Owners are expected to review code contributions, approve changes to this document, manage the copyrights within the project outputs, and participate in the project discussions and meetings.
+
+Owners fulfill all requirements of Peers, and also:
+
+- Manage and merge non-data pull requests such as schema, linter, or infrastructure changes.
+- May merge their own pull requests once they have collected the feedback they deem necessary. (No pull request should be merged without at least one peer or owner comment stating they’ve looked at the PR.)
+- Release a new npm version of the project on a regular (weekly) basis.
+
+To become an Owner one must fulfill at least the following conditions and commit to being a part of the community for the long-term.
+
+- Have worked in a helpful and collaborative way with the MDN community.
```

Comment:
This seems very like "team player" from https://github.com/mdn/browser-compat-data/pull/3668/files#diff-898e7f1ceacb493c024554f5a7c87bdfR39, and I like this wording better than "team player".

I like the idea of stressing "helpfulness to contributors" as a desirable attribute of both peers and owners. A big part of the role is interacting with people who have much less familiarity with the project and working with them to get their contributions merged.